lib/phase/functional.js

```diff
@@ -28,7 +28,9 @@ function validateJobMatrix(job, prefix) {
     const environment = Hoek.reach(job, 'environment', {
         default: {}
     });
-    const environmentSize = Object.keys(matrix).length + Object.keys(environment).length;
+    const SDEnv = ['SD_TEMPLATE_NAME', 'SD_TEMPLATE_VERSION'];
```

Comment:
Is there any way to convert this section into a method? Since it seems to duplicate logic done at line 116
Or at least declare these vars at a more global level